listenbrainz_spark/recommendations/candidate_sets.py

```diff
@@ -256,8 +249,8 @@ def main():
         sys.exit(-1)
     logging.info('Files fetched from HDFS and dataframes registered in {}s'.format('{:.2f}'.format(time() - ti)))
 
-    path = os.path.join(os.path.dirname(os.path.abspath(__file__)),'recommendation-metadata.json')
-    with open(path) as f:
+    _path = os.path.join(os.path.dirname(os.path.abspath(__file__)),'recommendation-metadata.json')
```

Comment:
variables starting with `_` are generally private variables in classes, just name it metadata_file_path?